ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -187,8 +187,58 @@ device_classes: !ZenPackSpec
                         datapoints:
                             status:
                                 rrdmin: 0
+                    diskstats:
+                        type: COMMAND
+                        commandTemplate: '/bin/cat /proc/diskstats'
+                        usessh: true
+                        parser: ZenPacks.zenoss.LinuxMonitor.parsers.linux.diskstats
+                        datapoints:
+                            totalReads:
```

Comment:
I have a couple of issues with the datapoints. I see all 11 fields from /proc/diskstats being captured. That's probably good, but most of them aren't being graphed. Any datapoint that's being stored should also be used in a graph unless there's a really good reason not to, such as the value only being useful to threshold, not to plot. These don't fit that rule.

I also think that some of these datapoints need different names now that they're rates (DERIVE)instead of totals (GAUGE).

* totalReads -> reads (readsCompleted?)
* numReadsMerged -> readsMerged
* totalSectorsRead -> sectorsRead
* totalMsReading -> msReading
* totalWrites -> writes (writesCompleted?)
* numWritesMerged -> writesMerged
* totalSectorsWritten -> sectorsWritten
* totalMsWriting -> msWriting
* numIOInProgress -> ioInProgress
* totalMsIO -> msDoingIO
* weightMsPerIO -> msDoingIOWeighted

It also looks like numIOInProgress/ioInProgress need to be a GAUGE_MIN_0. It's the one value that isn't a counter.